PAO-Project/src/Catalog/Catalog.java

```diff
@@ -0,0 +1,57 @@
+package Catalog;
```

Comment:
[Fix] toate pachete vin sub pachetul radacina, cel cu numele organizatiei/companiei pentru care creezi proiectul. In cazul asta este o companie fictiva, si folosim com.company. Am putea folosi com.example, sau ro.unibuc, etc